src/mugs.js

```diff
@@ -913,7 +913,8 @@ define([
             calculateAttr: { presence: "notallowed" },
             constraintAttr: { presence: "notallowed" },
             constraintMsgAttr: { presence: "notallowed" },
-            dataValue: { presence: "notallowed" }
+            dataValue: { presence: "notallowed" },
+            requiredAttr: { presence: "notallowed" },
```

Comment:
Completely agree in Python where it's allowed. However some JS (maybe only JSON?) parsers will choke on it. Doesn't seem worth much debate if tests pass and it works in Chrome and Firefox.